qemu/tests/cfg/qmp_command.cfg

```diff
@@ -62,9 +62,9 @@
             event_cmd_type = host_cmd
             qmp_cmd = "pmemsave val=0, size=4096, filename=/var/tmp/pmemsave"
             pre_cmd = "rm -rf /var/tmp/pmemsave"
-            post_cmd = "ls /var/tmp/pmemsave; true"
-            cmd_result_check = post_contain
-            cmd_return_value = pmemsave
+            post_cmd = "ls /var/tmp/pmemsave"
+            post_cmd_check = "stat -c %s /var/tmp/pmemsave"
+            cmd_result_check = "qmp_pmemsave"
```

Comment:
No need to use "post_cmd". 
Get file name and file size from "qmp_cmd". 
if os.path.exists(file_name):
    if size_output = file_size:
        ****